java/FoodTruckApp/src/main/java/com/my/FoodTruckApp/entree/EntreeRequestBody.java

```diff
@@ -1,15 +1,13 @@
 package com.my.FoodTruckApp.entree;
-
 import lombok.AllArgsConstructor;
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
-@AllArgsConstructor//annotation does the all args constructor
-@NoArgsConstructor//annotation does the no args constructor
-@Data //has all the getters, setters, and toString built in
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
 public class EntreeRequestBody {
-    private String entreeSize;
-    private String entreeFlavor;
-    private String entreePairedMeal;
-    private Integer entreePrice;
+    private Integer id;
```

Comment:
This is field should probably not be needed since it’s auto incremented right?

Reply:
You are right, I corrected it